modules/apps_api/app/workers/apps_api/fetch_connections.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require_relative '../../../lib/apps_api/notification_service.rb'
+
+module AppsApi
+  class FetchConnections
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def perform
+      notif_service = AppsApi::NotificationService.new
+      # process all connection events
+      notif_service.handle_event(
+        'app.oauth2.as.consent.grant',
+        Settings.vanotify.template_id.connection_template
```

Comment:
This should be `Settings.vanotify.lighthouse.template_id.connection_template`

And I believe @saman-moshafi-va is changing the Settings yaml structure to have 'templates' instead of 'template_id'.
So it will be:
`Settings.vanotify.lighthouse.templates.connection_template`
after his changes.

Sorry for all this confusion. It all is happening all at once...

Reply:
The entries for the lighthouse templates will be `Settings.vanotify.lighthouse.template_id.connection_template` and `Settings.vanotify.lighthouse.template_id.disconnection_template`.
PR 5998 will add the entries to settings.yml, once it is approved.

Reply:
Inside devops the vanotify block has `services`, but you're saying it's not going to?
```
vanotify:
  client_url: ...
  services:
    lighthouse:
      template_id:
        disconnection_template: ...
        connection_template: ...
```

So the template entry should be `Settings.vanotify.lighthouse.template_id.connection_template`, right?

Reply:
Correct, the entry should be Settings.vanotify.services.lighthouse.template_id.connection_template.  Devops is already changed to set the values for review, dev, staging and prod environments.

Reply:
Ok, so the references the the template_ids are correct and I'll mark this part resolved.